src/plugins/usage_collection/README.mdx

```diff
@@ -400,9 +402,9 @@ There are a few ways you can test that your usage collector is working properly.
 
 ## FAQ
 
-1. **How should I design my data model?**  
+1. **How should I design my data model?**
```

Comment:
Mind the spaces (it happened to me the other day, I think it's the MDX handler in prettier 😕 )

Reply:
I've been trying to get around that but still can't figure it out, it's super frustrating!
Should be ok now I hope